torch/_torch_docs.py

```diff
@@ -4649,6 +4649,33 @@ def merge_dicts(*dicts):
     torch.Size([3, 2, 1])
 """.format(**single_dim_common))
 
+add_docstr(torch.nanquantile,
+           r"""
+nanquantile(input, q, dim=None, keepdim=False, *, out=None) -> Tensor
+
+This is a variant of :func:`torch.quantile` that ignores `NaN` values when
```

Comment:
... that "ignores" NaN values, computing the quantiles :attr:\`q\` as if NaN values in :attr:\`input\` did not exist. See the documentation for :func:\`torch.quantile\` and the example below.

I don't think you need to mention when NaN is produced in the output, but if you do I think it's more consistent to use terminology like "if all values in a reduced row are NaN..."